src/applications/mhv-secure-messaging/components/AttachmentsList.jsx

```diff
@@ -178,9 +178,9 @@ const AttachmentsList = props => {
                           )}, button available: Remove ${file.name}`
                     }
                   >
-                    <i
-                      className="fas fa-paperclip"
-                      alt="Attachment icon"
+                    <va-icon
```

Comment:
Boilerplate icon inserted here

Reply:
It's really strange how all of these boilerplate icons have made their way into this pull request, especially since we removed that eslint quickfix last week :thinking:

Reply:
If this branch is up to date with the latest `main`, is it possible that those changes were already in there and wouldn't have been removed by the eslint fix?

Reply:
@randimays - yeah, I believe a lot of these were already in `main`. @powellkerry [created a PR](https://github.com/department-of-veterans-affairs/vets-website/pull/29662) into main that reverts these to font-awesome for the time being.